docs/zh_cn/juicefs_on_docker.md

```diff
@@ -1,2 +1,55 @@
 # Docker 使用 JuiceFS
 
+如果你希望使用 JuiceFS 为 Docker 容器提供持久化的存储，你需要先开启 FUSE 的 `user_allow_other` 选项，然后通过  `-o` 选项设置 `allow_other`重新挂载 JuiceFS 文件系统。
+
+## FUSE 设置
+
+### 修改配置文件
+
+编辑 FUSE 的配置文件，通常是 `/etc/fuse.conf`：
+
+```
+$ sudo nano /etc/fuse.conf
+```
+
+将配置文件中的 `user_allow_other` 前面的 `#` 注释符删掉，修改后类似下面这样：
+
+```
+# /etc/fuse.conf - Configuration file for Filesystem in Userspace (FUSE)
+
+# Set the maximum number of FUSE mounts allowed to non-root users.
+# The default is 1000.
+#mount_max = 1000
+
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
+```
+
+### 重新挂载 JuiceFS
+
+FUSE 的 `user_allow_other` 启用后，你需要重新挂载 JuiceFS 文件系统，使用 `-o` 选项设置 `allow_other`，例如：
+
+```
+$ juicefs mount -d -o allow_other redis://<your-redis-url>:6379/1 /mnt/jfs
```

Comment:
juicefs adds `allow_other` when the current user is root or using `sudo`

Reply:
When I mapping a sub-directory in JuiceFS mount point to a Docker container, I got the following output:
`2021-05-19 13:58:51.857830 I | Unimplemented opcode POLL`

FYI: My JuiceFS is composed with MinIO and MySQL, and I have enable user_allow_other in `/etc/fuse.conf`.

The command I am using to mount JuiceFS is:
```shell
$ sudo juicefs mount mysql://juicefs:xxxxxx@(localhost:3306)/juicefs juice-test/
```

And then I create a filebrowser container:
```shell
sudo docker run -d --name fb \
-p 8088:80 \
-v $PWD/fb/root:/srv \
-v $PWD/fb/database.db:/database.db \
filebrowser/filebrowser
```

After the container created, the app in container is running failed, Docker logs:
```
2021/05/19 13:57:46 bolt.Close(): funlock error: function not implemented
2021/05/19 13:57:46 function not implemented
```

But by the same way, I can create the container successful when I use a local directory.

Reply:
This one is OK.
```
2021-05-19 13:58:51.857830 I | Unimplemented opcode POLL
```

The SQL implementation does not support flock and fcntl, please use Redis in this case.